### ISSUE TITLE
STIL Test failed

### ISSUE THREAD

**Issue by Aaronself** [sbaby171/PySTIL#17]
Hello，I want to ask about the stil parser, which can not pass any test. If possible, can I ask if this software is the latest version?

**Comment by sbaby171** [comment]
Hey Aaronself, yeah this is old dead code. But am starting on new one. You need a STIL parser?

**Comment by Aaronself** [comment]
Yes, I need to refer to some stil analysis methods. I wonder if you can provide it here.

**Comment by sbaby171** [comment]
Okay, yeah, I am working on a better one currently. What are some of the analysis methods you are aiming to do  (what data do you need to analyze from the STILs) also do you know that general size of the STIL files you will be reading? Only asking to see if there are huge STIL files

**Comment by rbarzic** [comment]
@sbaby171 : Any update about this "new" version of the STIL parser ?

**Comment by sbaby171** [comment]
@rbarzic  -- I have looked at STILs since these posts. I am happy to come back to this but need some input on what is needed
